doc/Programming-Guide/Groups.dox

```diff
@@ -33,7 +33,7 @@
  *   All requests to origin-server are sent in that servers protocol.
  *
  \par
- *   Wais and Gopher don't receive much
+ *   Wais and Whois don't receive much
```

Comment:
This is wrong. Whois is an ACL lookup and still frequently used by major proxies.

Reply:
Yes, thank you.